presto-main/src/main/java/com/facebook/presto/sql/relational/FunctionResolution.java

```diff
@@ -230,12 +230,12 @@ public boolean isSubscriptFunction(FunctionHandle functionHandle)
 
     public FunctionHandle tryFunction(Type returnType)
     {
-        return functionManager.lookupFunction("TRY", fromTypes(returnType));
```

Comment:
I don't get this change. If `TRY` is registered as `$internal$try`, how could this work before? Otherwise how could this work after?

Reply:
We never called this before. Try will always be desugared into a FunctionCall in `DesugarTryExpression`. So we will never actually call `visitTry` in `SqlToRowExpressionTranslator`.